app/javascript/components/Results/ResultGrid/ResultGrid.tsx

```diff
@@ -84,6 +85,13 @@ export const ResultGrid = ({ result, affiliate, query, position, vertical }: Res
           <div className='result-desc'>
             {getDescription(result.description)}
             <div className='result-url-text'>{truncateUrl(result.url, URL_LENGTH)}</div>
+            {/* Filter tags UI - to be updated with integration once backend starts sending the data */}
+            {facetsEnabled && 
+            <div className='filter-tags-wrapper'>
+              <span className='filter-tag'>Small Business</span>
```

Comment:
This should be an iteration over an array. Let's hardocode filter tags on the response as if they would come from the backend and iterate over them here.

Reply:
Code is updated to get the filter tags from a function. This will be updated to read the dynamic values once backend starts sending the data.